iFixit/react-commerce
# Routing: Exclude `/Store` prefix from urls

All `/Store/Parts/*` pages are now at `/Parts/*`. Same for `/Tools`.

`Store/[handle]` pages still exist. We'll redirect them to `/Tools/[handle]` or `/Parts/[handle]` in the future, but not in this pull.

## QA
Visit the preview. Make sure that there are no more /Store/Parts/* pages and no more /Store/Tools/* pages. Make sure we don't link to those kinds of pages anywhere either. Thanks!

CC @sterlinghirsh 

Closes #322
